de.blobbyvolley.BlobbyVolley2.yaml

```diff
@@ -0,0 +1,65 @@
+app-id: de.blobbyvolley.BlobbyVolley2
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: blobby
+
+finish-args:
+  - --device=dri
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --persist=.blobby
```

Comment:
Can't it use the xdg configs dir automatically? or at least patched to do so?

Reply:
I created issue for this but I would like to avoid more patches.

https://github.com/danielknobe/blobbyvolley2/issues/15